horace_core/sqw/@axes_block/build_from_input_binning.m

```diff
@@ -87,6 +87,20 @@
             range = [range(1),range(3)];
         end
     end
+    if numel(range) == 2
+        if range(1)==range(2)  % some old file formats do not store proper img_range
+            % and store 0,0 for integration ranges. Here we try to mitigate
+            % this
+            av_pt = 0.5*(range(1)+range(2));
```

Comment:
Assuming your comment is correct and the case you are trying to catch is range==[0,0], av_pt==0. Even if not, av_pt==range(1). May be worth tidying this up on this basis.